bigtable/client/table_readrow_test.cc

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/table.h"
+#include "bigtable/client/testing/table_test_fixture.h"
+
+namespace {
+class MockResponseStream : public grpc::ClientReaderInterface<
```

Comment:
maybe move to table_test_fixture.h, since it's the same with the one in table_readrows_test?

Reply:
Good idea.  Will refactor before merging.